.github/workflows/aggregate.yaml

```diff
@@ -41,3 +40,13 @@ jobs:
       - name: Publish output
         run: ardrive upload-file --local-path ./daily_output.json -F "d62a47c3-0b9d-4442-ac72-252a239d0469" -w /tmp/wallet --no-bundle --upsert
 
+      - name: Commit GQL cache
+        if: always()
+        uses: github-actions-x/commit@v2.9
```

Comment:
This version doesn't work anymore. We should use actions-x/commit@v6
like this:
```- name: Commit GQL cache
      uses: actions-x/commit@v6
      with:
        name: GitHub Actions Cache Committer
        branch: production
        files: cache
```

Reply:
Nice catch! 😎 
5fdbae3e9455c4837b32b9718ba2acab6b3e1aac